app/js/components/LabTrendsPage.jsx

```diff
@@ -133,12 +134,18 @@ export class LabTrendsPage extends PureComponent {
       bottom: 5,
     };
 
-    const chartData = R.compose(
```

Comment:
Compose processed right to left, which was confusing, I changed this to a pipe with operates left-to-right.